src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_ISymbolInitializer.cs

```diff
@@ -427,5 +427,1539 @@ class C
+          Left: 
+            IPropertyReferenceOperation: System.Int32 C.P1 { get; } (Static) (OperationKind.PropertyReference, Type: System.Int32, IsImplicit) (Syntax: '= M()')
+              Instance Receiver: 
+                null
+          Right: 
+            IInvocationOperation (System.Int32 C.M()) (OperationKind.Invocation, Type: System.Int32) (Syntax: 'M()')
+              Instance Receiver: 
+                null
+              Arguments(0)
+
+    Next (Regular) Block[B2]
+Block[B2] - Exit
+    Predecessors: [B1]
+    Statements (0)
+";
+            var expectedDiagnostics = DiagnosticDescription.None;
+
+            VerifyFlowGraphAndDiagnosticsForTest<EqualsValueClauseSyntax>(source, expectedFlowGraph, expectedDiagnostics);
+        }
+
+        [CompilerTrait(CompilerFeature.IOperation, CompilerFeature.Dataflow)]
+        [Fact]
+        public void NoControlFlow_NonConstantPropertyInitializer_InstanceProperty()
+        {
+            string source = @"
+class C
+{
+    public int P1 { get; } /*<bind>*/= M()/*</bind>*/;
+    public static int M() => 0;
+}
+";
+            string expectedFlowGraph = @"
+Block[B0] - Entry
+    Statements (0)
+    Next (Regular) Block[B1]
+Block[B1] - Block
+    Predecessors: [B0]
+    Statements (1)
+        ISimpleAssignmentOperation (OperationKind.SimpleAssignment, Type: System.Int32, IsImplicit) (Syntax: '= M()')
+          Left: 
+            IPropertyReferenceOperation: System.Int32 C.P1 { get; } (OperationKind.PropertyReference, Type: System.Int32, IsImplicit) (Syntax: '= M()')
+              Instance Receiver: 
+                IInstanceReferenceOperation (ReferenceKind: ContainingTypeInstance) (OperationKind.InstanceReference, Type: C, IsImplicit) (Syntax: '= M()')
+          Right: 
+            IInvocationOperation (System.Int32 C.M()) (OperationKind.Invocation, Type: System.Int32) (Syntax: 'M()')
+              Instance Receiver: 
+                null
+              Arguments(0)
+
+    Next (Regular) Block[B2]
+Block[B2] - Exit
+    Predecessors: [B1]
+    Statements (0)
+";
+            var expectedDiagnostics = DiagnosticDescription.None;
+
+            VerifyFlowGraphAndDiagnosticsForTest<EqualsValueClauseSyntax>(source, expectedFlowGraph, expectedDiagnostics);
+        }
+
+        [CompilerTrait(CompilerFeature.IOperation, CompilerFeature.Dataflow)]
+        [Fact]
+        public void ControlFlow_NonConstantPropertyInitializer_StaticProperty()
+        {
+            string source = @"
+class C
+{
+    public static int P1 { get; } /*<bind>*/= M() ?? M2()/*</bind>*/;
+    public static int? M() => 0;
+    public static int M2() => 0;
+}
+";
+            string expectedFlowGraph = @"
+Block[B0] - Entry
+    Statements (0)
+    Next (Regular) Block[B1]
+Block[B1] - Block
+    Predecessors: [B0]
+    Statements (1)
+        IFlowCaptureOperation: 0 (OperationKind.FlowCapture, Type: null, IsImplicit) (Syntax: 'M()')
```

Comment:
>null [](start = 67, length = 4)

I didn't understand why the type in the flow capture is null. Is it ever expected to be non-null?

Reply:
> I didn't understand why the type in the flow capture is null. Is it ever expected to be non-null?

This node doesn't represent a value, nodes like that are not supposed to have types

---
In reply to: [187696597](https://github.com/dotnet/roslyn/pull/26763#discussion_r187696597) [](ancestors = 187696597)